### PR TITLE
[Login - Backend] Python Configurations Updates + Package Versions Updates

### PR DESCRIPTION
**Pull request by fnormand01**
# Description

This PR is updating the Python Configurations, meaning putting back some of the useful configurations that we had before this major back end update. This PR is also upgrading the package versions to their latest compatible ones.

**Pipfile** and **Pipfile.lock** files have also been deleted in this PR. These file were generated due to the virtual environment I had before for testing.

**Known Vulnerability Issue**
---
image

**More about this vulnerability:** https://snyk.io/vuln/SNYK-PYTHON-DJOSER-174565

## Type of change

- Code cleanliness or refactor
- Dependency change

## Screenshot

(N/A)

# Testing

Manual steps to reproduce this functionality:

- Just make sure that everything is working as expected, since there is no visual or behavior changes.

# Checklist

Applicable for all code changes.

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new compiler warnings
- [ ] My changes generate no new accessibility errors and/or warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)
- [ ] My changes look good on IE 11+ and Chrome

### CODE REVIEW

**Comment by fnormand01** [comment]
@MichaelCherry, to answer your questions, these version updates are just upgrading all the possible version to the latest ones (if possible), since the login example project is one year old. Also, yes I deleted the production configs, since we don't have any production environment yet. We'll add it once we'll be there.

File: backend/config/environment.py [hunk 1]
```diff
@@ -1,12 +1,9 @@
 ENVIRONMENT = "local"
 # ENVIRONMENT = "development"
-# ENVIRONMENT = "production"
 
 SETTINGS_MODULE = "config.settings.local"
 
 if ENVIRONMENT == "local":
     SETTINGS_MODULE = "config.settings.local"
 if ENVIRONMENT == "development":
```
Comment (caleybrock) [inline backend/config/environment.py]: I believe elsewhere we use "dev" as an environment variable.

File: backend/config/settings/development.py [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
 
-# Used for Debug Toolbar to show
-INTERNAL_IPS = ["127.0.0.1"]
-
 # TODO: Setup the real dev DB here
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+        "ENGINE": "django.db.backends.postgresql",
```
Comment (caleybrock) [inline backend/config/settings/development.py]: These are the settings for our local DB.

Comment (caleybrock) [reply backend/config/settings/development.py]: The dev DB should be set using the environment variables like we have in our current config.

**Comment by caleybrock** [comment]
I think it might be best to merge these changes (into your other branch) and update them in your other PR.

It's hard to review because I want to compare the config to what we currently have, not what you did in the other PR.

File: backend/config/environment.py (hunk 1 above)
Comment (caleybrock) [reply backend/config/environment.py]: Else should be the local so that local is always default.

**Pull request merged**